DLL/Irdroid/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8)
+
+# Use MFC in a shared DLL.
+set(CMAKE_MFC_FLAG 2)
```

Comment:
You're not using MFC. Make that 0 instead of 2.

Reply:
Ok, I did the required changes
Thanks.